tests/cpp/CMakeLists.txt

```diff
@@ -15,11 +15,6 @@ if(WITH_TESTING)
       continue() # the unittest for gemm requires extra dependencies
     endif()
 
-    # FIXME(haruhi): recover these commented unittest after refactor.
```

Comment:
Can we re-add it now?

Reply:
This PR add the `test_s2r_copy` and pass the unittest.

There are other two unittests being deleted, test a single wmma, and test_gemm will be added in the next two PRs.